crates/ruma-common/src/events/secret_storage/key.rs

```diff
@@ -0,0 +1,121 @@
+//! Types for the *m.secret_storage.key.\** event.
+
+use js_int::{uint, UInt};
+use ruma_common::identifiers::{KeyDerivationAlgorithm, SecretEncryptionAlgorithm};
```

Comment:
```suggestion
use crate::identifiers::{KeyDerivationAlgorithm, SecretEncryptionAlgorithm};
```

and should be moved below to the other crate import.